tests/helpers/aupro_reference.py

```diff
@@ -0,0 +1,221 @@
+# REMARK: CODE WAS TAKEN FROM https://github.com/eliahuhorwitz/3D-ADS/blob/main/utils/au_pro_util.py
```

Comment:
We should also add a license file next to this similar to https://github.com/openvinotoolkit/anomalib/blob/development/anomalib/models/draem/LICENSE. This should include a copy of the license contained in https://www.mydrive.ch/shares/45924/9ce7a138c69bbd4c8d648b72151f839d/download/428846918-1643297332/evaluation_code.tar.xz. However, I am more concerned about this repo https://github.com/eliahuhorwitz/3D-ADS as it does not have a license. Maybe it would be a good idea to request the authors to add a license to their repo.

Reply:
I added the required licenses, but please do double check.